p2p/net/connmgr/connmgr_test.go

```diff
@@ -520,7 +520,7 @@ func TestPeerProtectionSingleTag(t *testing.T) {
 	}
 
 	// protect the first 5 peers.
-	var protected []network.Conn
+	protected := make([]network.Conn, 0, len(conns[0:5]))
```

Comment:
Let's not do this, this looks error-prone. This is a test anyway, so I wouldn't care about allocs.